validate/validate_test.go

```diff
@@ -417,7 +415,7 @@ func TestValidateSnpAttestation(t *testing.T) {
 			name:        "rejected provisional by tcb",
 			attestation: attestationcb1455,
 			opts:        &Options{ReportData: noncecb1455[:], GuestPolicy: abi.SnpPolicy{Debug: true}},
-			wantErr:     "the report's COMMITTED_TCB 0x9270000000007f00 does not match the report's CURRENT_TCB 0x9270000000007f1f",
+			wantErr:     "the report's COMMITTED_TCB &{0 127 0 0 0 0 112 146} does not match the report's CURRENT_TCB &{31 127 0 0 0 0 112 146} (<nil>, <nil>)",
```

Comment:
This is less readable to someone trying to interpret the error directly from SEV-SNP API documentation.